### PR TITLE
Timothy Hang Lunch Order

### PR DESCRIPTION
**Pull request by TimE1211**

### CODE REVIEW

File: bamboozle.md [hunk 1]
```diff
@@ -17,5 +17,6 @@ Developer     | Order
 [Brad](https://github.com/bself)                    | Beef Banh Mi w/Spicy Mayo on the side
 [Minho](https://github.com/minhochoi)               | Large Chicken & Rice Soup and 1 grilled beef rolls (soy sauce)
 [Kyle](https://github.com/kjswartz)                 | Small Chicken & Rice Soup and 1x Haas Avocado roll: lettuce, carrots, mango 1x peanut sauce
+[Tim](https://github.com/TimE1211)                  | gimme some of that water o baby
```
Comment (jessecurry) [inline bamboozle.md]: @TimE1211 is this a typo or did you just want water? If it's just water I'll leave this PR out so we don't confuse the restaurant.

**Comment by TimE1211** [comment]
Oops that was the first draft I made, sorry. Let me update that really quick.

**Comment by TimE1211** [comment]
I thought I pushed with the final commit sorry, I just had to redo it

**Comment by jessecurry** [comment]
No worries, it's all merged now. See you tomorrow!

**Pull request merged**